magento/magento2
# honor queue_consumer.xml maxMessages

### Description (*)


The maxMessages="100" attribute in `app/code/Magento/MediaStorage/etc/queue_consumer.xml` for instance is not being used. Instead only deployment configuration is used.

### Related Pull Requests


### Fixed Issues (if relevant)


### Manual testing scenarios (*)

1. install vanilla Magento and run `pgrep -fl queue:consumers:start`
2. see that all consumers have `--max-messages=10000`
3. apply the PR
4. run `pkill -f queue:consumers:start`
5. wait for cron to restart the consumers
6. see that the values from queue_consumer.xml are used with a fallback and max of the deployment config or 10000 if that is not set.

### Questions or comments


### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#29522: honor queue_consumer.xml maxMessages
